demisto_sdk/commands/lint/lint_manager.py

```diff
@@ -74,24 +76,36 @@ def __init__(self, input: str, git: bool, all_packs: bool, quiet: bool, verbose:
                                                          all_packs=all_packs,
                                                          base_branch=self._prev_ver)
 
-        self._id_set_path = id_set_path
         if check_dependent_api_module:
-            print('Checking for packages dependent on the modified API module', end='... ')
-            dependent_on_api_module = get_api_module_dependencies(self._pkgs, self._id_set_path, self._verbose)
-            dependent_on_api_module = self._get_packages(content_repo=self._facts["content_repo"],
-                                                         input=dependent_on_api_module)
+            dependent_on_api_module = self._get_api_module_dependent_items()
             self._pkgs = list(set(self._pkgs + dependent_on_api_module))
-            if dependent_on_api_module:
-                print(f'Found {Colors.Fg.cyan}{len(dependent_on_api_module)}{Colors.reset} dependent packages. '
-                      f'Executing lint and test on those as well.')
-            else:
-                print('No dependent packages found.')
+
         if json_file_path:
             if os.path.isdir(json_file_path):
                 json_file_path = os.path.join(json_file_path, 'lint_outputs.json')
         self.json_file_path = json_file_path
         self.linters_error_list: list = []
 
+    def _get_api_module_dependent_items(self) -> list:
+        changed_api_modules = {pkg.name for pkg in self._pkgs if API_MODULES_PACK in pkg.parts}
+        if changed_api_modules:
+            dependent_items = []
+            for changed_api_module in changed_api_modules:
+                print(f'Checking for packages dependent on the modified API module {changed_api_module}...')
+                with Neo4jContentGraphInterface() as graph:
+                    api_module_node = graph.search(object_id=changed_api_module)[0]
```

Comment:
I think we can't assume there is a built graph (locally / on circle)
@ilaner let's discuss this tomorrow - maybe we need an inner check for this

Reply:
anyway, please wrap it in a try except clause, and handle a case which the object_id is not found the graph

Reply:
@dantavori @ilaner Why can't we assume the graph is built?

I think this step should fail if the graph is not updated/created. That way we won't pass failing modules that import the api modules.

I raised an exception for an out of date graph file & I don't see sense in try excepting the graph errors just to raise them again, unless you say they are unreadable in which case I can add a print indicating the errors came from trying to interact with the content graph